README.md

```diff
@@ -1 +1,70 @@
-# facebookClone-frontend
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
Could you please change the readme file back to as it was "# facebookClone-frontend" instead of the default generated readme by react